ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py

```diff
@@ -1557,3 +1557,46 @@ def test_torch_cov(
         fweights=fweights,
         aweights=aweights,
     )
+
+
+# bucketize
+@handle_frontend_test(
+    fn_tree="torch.bucketize",
+    dtypes_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=0,
+    ),
+    dtypes_and_boundaries=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1,
+        max_num_dims=1,
+    ),
+    out_int32=st.booleans(),
+    right=st.booleans(),
+)
+def test_torch_bucketize(
+    dtypes_and_input,
+    dtypes_and_boundaries,
+    out_int32,
+    right,
+    test_flags,
+    backend_fw,
+    on_device,
+    frontend,
+    fn_tree,
+):
+    dtypes_input, input = dtypes_and_input
+    dtypes_boundaries, boundaries = dtypes_and_boundaries
+    boundaries[0] = np.sort(boundaries[0])

```

Comment:
This line might be causing the issue, modify this to `np.sort(boundaries[0], axis=0)`

Reply:
Thanks for the quick reply. I tried that and unfortunately the tests still fail. Printing `boundaries` I can see that sometimes arrays of dimension <1 and >1 are produced, e.g.
```
boundaries: [array(6, dtype=uint8)]
boundaries: [array([[[[[-6.9198933e+16],
          [ 1.9000000e+00]]],


        [[[ 5.9604645e-08],
          [ 1.0000000e+00]]],


        [[[ 1.1920929e-07],
          [-3.0934760e+38]]],


        [[[-2.0000100e+00],
          [-7.3511812e+15]]],


        [[[-3.0934760e+38],
          [-1.0000000e+00]]],


        [[[-6.5933984e+16],
          [-5.9604645e-08]]],


        [[[ 2.0000100e+00],
          [-5.9604645e-08]]],


        [[[ 1.3810570e+37],
          [ 1.4916630e+16]]],


        [[[ 6.1035156e-05],
          [-2.0000100e+00]]],


        [[[ 3.3333334e-01],
          [ 2.1191212e+38]]]]], dtype=float32)]
```
I believe this is to do with `helpers.dtype_and_values`. Am I misunderstanding the function? I assumed that by setting `min_num_dims=1` and `max_num_dims=1` I would only ever get arrays of dimension 1.

Reply:
Currently, the issue is raised due to not setting axis=0, can you do the change for this so that we could look into the exception which you mentioned.

Reply:
I added `axis=0`